app/renderer/js/utils/link-util.js

```diff
@@ -19,7 +19,11 @@ class LinkUtil {
 		const currentDomain = wurl('hostname', currentUrl);
 		const newDomain = wurl('hostname', newUrl);
 
-		return (currentDomain === newDomain) && newUrl.includes('/#narrow');
+		const sameDomainUrl = (currentDomain === newDomain || newUrl === currentUrl + '/');
+		const isInternalUrl = newUrl.includes('/#narrow') ||
+      newUrl.includes(currentUrl + '/user_uploads/');
```

Comment:
@cPhost do we need this here? Since we already added this in the `handle-external-link.js`.

Reply:
I think i left it here since i think it makes sense here, ie if we use this somewhere else this should be standalone

Reply:
`isInternal` sound like it should support this so, i am tempted to add it here.